src/consensus/consensus.rs

```diff
@@ -75,8 +75,8 @@ impl Default for Config {
             private_key: hex::encode(SecretKey::generate()),
             shard_ids: vec![1],
             num_shards: 1,
-            propose_value_delay: Duration::from_millis(250),
-            max_messages_per_block: 250, //TODO
+            block_time: Duration::from_millis(250),
+            max_messages_per_block: 500,
```

Comment:
~1mbs at 200 bytes per messages * 500 * 2 shards * 4 blocks per sec at 250ms. 

We probably have some room to bump this higher, but need to be mindful of the number of shards running on the same node.